packages/i18n/src/messages/errorMessages.ts

```diff
@@ -47,6 +47,12 @@ const errorMessages = defineMessages({
     description:
       "Error Message displayed on word counter when user passes the limit.",
   },
+  overCharacterLimit: {
+    defaultMessage: "This field must have less than {value} characters.",
```

Comment:
In case this is the only translation needed, here is a suggestion: 

```
Ce champ doit contenir moins de {value} caractères.
``` 

We already have a nearly identical string for this purpose:

https://github.com/GCTC-NTGC/gc-digital-talent/blob/62cbc97fa2e78ffda23f9a586b87226915780adf/packages/i18n/src/messages/errorMessages.ts#L44-L49

https://github.com/GCTC-NTGC/gc-digital-talent/blob/62cbc97fa2e78ffda23f9a586b87226915780adf/packages/i18n/src/lang/fr.json#L1830-L1833


Also, [Termium](https://www.btb.termiumplus.gc.ca/tpv2alpha/alpha-eng.html?lang=eng&i=1&srchtxt=CHARACTER&codom2nd_wet=1&index=alt) suggests _caractère_ as the translation for this type of **character**.

Reply:
Hm, I think that is reasonable, almost identical with the exception of the term at the end and we probably want these two translations in sync anyway since their purpose/phrasing is the same

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/pull/8830/commits/1fbf0da939ad61d83c93c2efca980c654f3b149a